hardware/products/ucm/intro.md

```diff
@@ -1,3 +1,249 @@
-# UCM Products Introduction
+# UCM Introduction
 
-Coming soon.
\ No newline at end of file
+What is UCM? UCM is a Universal Compute Module based on the 200-pin SO-DIMM, Small Outline Dual In-line Memory Module. To make these modules mostly interchangeable, GHI Electronics has made a standard pin format for these 200-pins.
+
+The standard consists of several widely used peripherals found on many microcontrollers such as UART, SPI, I2C, PWM, etc.
+
+The peripherals found on each module may vary.
+# UCM Standard
+## UCM Standard - Available Peripherals
+| Peripheral											| Up to Max	|
+| :----------------------------------------------------	| :--------	|
+| UART (Universal Asynchronous Receiver/Transmitter)	| 4			|
+| UART HS (Handshaking)									| 2			|
```

Comment:
Expand this to clarify what exact pins it has, that it's UART with handshaking, not just handshaking pins.